tools/geoipdatabases/main.go

```diff
@@ -0,0 +1,32 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+// Tool based on the code at https://github.com/maxmind/MaxMind-DB
+// (commit: https://github.com/maxmind/MaxMind-DB/commit/0ec71808b19669e9e1bf5e63a8c83b202d9bd115)
+
+func main() {
+	source := flag.String("source", "", "Source data directory")
+	target := flag.String("target", "", "Destination directory for the generated mmdb files")
```

Comment:
Nit. We could hard-code these paths to the paths we have.

Reply:
Done in https://github.com/elastic/elastic-package/pull/2450/commits/f6799e449bb2f101ca83c5642d24f22dd6deb01a
and updated README accordingly.